lib/screens_web/controllers/v2/screen_controller.ex

```diff
@@ -76,6 +76,16 @@ defmodule ScreensWeb.V2.ScreenController do
     |> render("index_multi.html")
   end
 
+  def widget(conn, %{"id" => app_id} = params)
```

Comment:
Need to move this def either down or up in the file so that it doesn't come between the clauses of `index/2` (per the compile error message in most recent CI run)